ignition/README.md

```diff
@@ -1,23 +1,25 @@
 # The configuration of the Ignition network
 
 ## Chain config
-- The `ChainId` is `0`.
-- The initial `privileged_address` which can perform the network upgrade: b62c6f41c4c380d78ae67a8b432a4a41e2465383f8cc9869b67106835685c388
-- The public address of the authority node that produces blocks: e8df6d432b2584a3ea9d0badf297c3b525bae71b577d22c1ccc12519adb64d92
+- The `ChainId` is `9889`.
+- The initial `privileged_address` which can perform the network upgrade: `eb42c979046d091f4f524e1efd8d4ffad92629b9a506b268a511861b0dbf2366`
+- The genesis public address of the authority node that produces blocks: `da6b021bf570f126869591f2009a95efb826f69272987db699fdea1de813697e`
 - The block gas limit is `30000000`.
+- Genesis state transition version is set to `10` - corresponding release is `fuel-core 0.35.0`.
 
 ### Gas costs
 
-The gas costs was created from the [benchmarks_fuel_core_0_26_0.json](benchmarks_fuel_core_0_26_0.json) benchmark results.
-The `new_storage_per_byte` is manually set to be `63`.
-The `vm_initialization.gas_per_unit` is manually set to be `0`.
+The gas costs was created from the [benchmarks_fuel_core_0_35_0.json](benchmarks_fuel_core_0_35_0.json) benchmark results.
+The `gas_per_byte` is manually set to be `233`.
```

Comment:
I set the `gas_per_byte` to be `233` instead of `63` until https://github.com/FuelLabs/fuel-core/issues/2133 is implemented. It means that the transaction can be a maximum of 126KB, which, guarantees the block is not more than 128 KB.

<img width="444" alt="image" src="https://github.com/user-attachments/assets/91f37646-65dd-4a8c-8619-1eef68c8ebd6">

Reply:
How would this affect the prices of common transactions? While it's one way to ensure we don't exceed the max block size, could this lead to overcharging of the common / typical transactions?

Reply:
Yes, it will make all transaction 4 times more expensive. But I hope before public phase we will land https://github.com/FuelLabs/fuel-core/issues/2133